.spelling

```diff
@@ -857,6 +857,13 @@ bs
 # To allow Site Init
 Init
 
+- operations/observability/Observability.md
+IaC
+KPIs
+K8s
+Systemd
+LiveDVD
```

Comment:
LiveDVD is not correct, the right term is LiveCD.

Systemd should be `systemd`, and "K8s" should be Kubernetes.

Reply:
I'm porting over a few of the lint changes from #3140.

Reply:
Sure, let me make these changes.

Reply:
@mitcharf, @nrockershousen, and @deantroe you may want to look over this PR for your own lint/corrections. This content has attempted to merge a few times, and some of the changes each pull-request brought in are no longer present.

Full list of PRs:
- github.com/Cray-HPE/docs-csm/pull/3140 <-- Had feedback from @mitcharf, @nrockershousen,  and @deantroe 
- github.com/Cray-HPE/docs-csm/pull/3359 
- github.com/Cray-HPE/docs-csm/pull/3305
- github.com/Cray-HPE/docs-csm/pull/3288
- github.com/Cray-HPE/docs-csm/pull/3036


Nathan is out of the office this week, I'll compare his feedback from #3140 to this PR (#3359) and add it accordingly.

Reply:
@rustydb I have made the changes requested - LiveCD, `systemd`, and Kubernetes are used instead of old entries. Removed these from the .spelling as well. Checks are successful.